test/unit/trace.test.js

```diff
@@ -351,6 +354,53 @@ describe('Trace', function() {
     expect(events.length).to.equal(0)
   })
 
+  it('parent.* attributes should be present on generated spans', function() {
+    // Setup DT
+    const encKey = 'gringletoes'
+    agent.config.encoding_key = encKey
+    agent.config.attributes.enabled = true
+    agent.config.distributed_tracing.enabled = true
+    agent.config.trusted_account_key = 111
+
+    const dtInfo = {
+      ty: 'App',       // type
+      ac: 111,         // accountId
+      ap: 222,         // appId
+      tx: 333,         // transactionId
+      tr: 444,         // traceId
+      pr: 1,           // priority
+      sa: true,        // sampled
+      // timestamp, if in the future, duration will always be 0
+      ti: Date.now() + 10000
+    }
+    const dtPayload = new DTPayload(dtInfo)
+    const headers = { [NEWRELIC_TRACE_HEADER]: dtPayload.httpSafe() }
+    const transaction = new Transaction(agent)
+    transaction.sampled = true
+
+    // Get the parent attributes on the transaction
+    transaction.acceptDistributedTraceHeaders('HTTP', headers)
+
+    // Create at least one segment
+    const trace = new Trace(transaction)
+    const child = transaction.baseSegment = trace.add('test')
+    child.start()
+    child.end()
+
+    // This should add the parent attributes onto a child span event
+    trace.generateSpanEvents()
+
+    // Test that a child span event has the attributes
+    const attrs = child.attributes.get(DESTINATIONS.SPAN_EVENT)
```

Comment:
Given this PR is introducing these attributes being added to spans, I'd recommend verifying it made it to span that was generated via `trace.generateSpanEvents()` instead of grabbing off the segment. While we know that destination *should* result in the right thing... we should have the Span available so might as well fully check working.